Source/WarpX.cpp

```diff
@@ -554,6 +554,10 @@ WarpX::ReadParameters ()
         }
 
         pp.query("do_pml", do_pml);
+        pp.query("do_silver_mueller", do_silver_mueller);
+        if ( (do_pml==1)&&(do_silver_mueller==1) ) {
+            amrex::Abort("PML and Silver-Mueller boundary conditions cannot be activated at the same time.");
```

Comment:
do we also need an abort for turn on Siler-Mueller with num_levels>1

Reply:
yes, we don't do this generally yet in the code when the mesh refinement patch touches the global boundary. We need to fix this for all boundary conditions in a separate issue.